include/revng/Support/GraphAlgorithms.h

```diff
@@ -243,21 +243,27 @@ template<class GraphT, class GT = llvm::GraphTraits<GraphT>>
 llvm::SmallSetVector<revng::detail::EdgeDescriptor<typename GT::NodeRef>, 4>
 getBackedges(GraphT Block) {
   llvm::SmallPtrSet<typename GT::NodeRef, 4> EmptySet;
-  return getBackedgesImpl<revng::detail::FilterSet::BlackList>(Block, EmptySet);
+  return getBackedgesImpl<revng::detail::FilterSet::BlackList,
+                          GraphT,
+                          GT>(Block, EmptySet);
```

Comment:
Oh nice catch. So we were computing the backedges on the wrong graph because of the missing template arguments?